README.md

```diff
@@ -59,7 +59,7 @@ have access to the installed PyQt4.
 To start Orange GUI from the command line, assuming it was successfully
 installed, run:
 
-    orange
+    orange-canvas
```

Comment:
Indeed there's a prior application, but that application is unrelease and unmaintained since 2009. I don't know anyone that uses it, for some measure. :smiley:

It's been [dropped from Debian in 2013](https://packages.qa.debian.org/o/orange/news/20131027T125412Z.html) for lack of maintenance, and it only exists in oldstable [wheezy](https://packages.debian.org/search?keywords=orange) where it will remain eternal. A similar package is still available in Fedora and SUSE, but I find this an unfortunate coincidence. No one is using that package or running that binary, surely. :stuck_out_tongue: 

What practical problems do you envision ahead if we just go with `orange`?